tox.ini

```diff
@@ -16,5 +16,4 @@ commands =
 
 [textenv:quality]
 commands =
-    pycodestyle edx_notifications
-    pylint edx_notifications --reports=no
+    make quality
```

Comment:
I would flip this around the other way.  `make quality` should run `tox -e quality`